src/controllers/buyerReview.ts

```diff
@@ -62,11 +62,11 @@ export const getBuyerReviewsByBuyerUser = async (buyer_id: string) => {
   }
 };
 
-export const getBuyerReviewsByPostingUser = async (user_id: string) => {
```

Comment:
A route for BuyerReview (api/reviews/buyer/byuser/:userId) still uses the function